api/services/ocean/aws/schemas/ocean-compute.yaml

```diff
@@ -38,14 +40,16 @@ properties:
           type: string
   launchSpecification:
     type: object
+    description: |
+      A subset of nodes on the cluster that you can configure for a specific purpose. Referred to in Spot as a virtual node group (VNG).
```

Comment:
actually in the ocean object level that's not the message that we want to convey, but that:
"The default specification for nodes in the cluster. Used as a template for custom Virtual Node Groups to inherit values which are not explicitly defined in them, and as a default VNG when no other VNGs exist or match the workload’s requirements." @netapp-jmontel

Reply:
Got it. Thanks.